classes/class-swsales-sitewide-sale.php

```diff
@@ -527,7 +557,26 @@ public function __get( $key ) {
 	 * @return boolean
 	 */
 	public function is_running() {
-		return ( $this->is_active_sitewide_sale() && 'sale' === $this->get_time_period() );
+		// Don't check if the sale is hidden in the admin.
+		if ( is_admin() ) {
+			return ( $this->is_active_sitewide_sale() && 'sale' === $this->get_time_period() );
+		}
+
+		// Allow admins to preview the sale period and banners regardless of whether sale is hidden using a URL attribute.
+		if ( current_user_can( 'administrator' ) && isset( $_REQUEST['swsales_preview_time_period'] ) || isset( $_REQUEST['swsales_preview_sale_banner'] ) ) {
+			return ( $this->is_active_sitewide_sale() && 'sale' === $this->get_time_period() );
+		}
+
+		// Check whether the sale should be hidden for the current user.
+		$hide = $this->hide_sale();
+		
+		// If the sale is hidden for this user, return.
+		if ( $hide ) {
+			return;
+		} else {
+			return ( $this->is_active_sitewide_sale() && 'sale' === $this->get_time_period() );
+		}
```

Comment:
```suggestion
		}

		return ( $this->is_active_sitewide_sale() && 'sale' === $this->get_time_period() );
```
Follows pattern from above code